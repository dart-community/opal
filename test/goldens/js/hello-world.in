const message = 'Hello, world!';

// Print the message.
console.log(`The message is: ${message}`);
