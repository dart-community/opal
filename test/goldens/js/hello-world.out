const message = 'Hello, world!';
^^^^^
source-code-js-keyword-const
     ^
     source-code-js-whitespace
      ^^^^^^^
      source-code-js-identifier
             ^
             source-code-js-whitespace
              ^
              source-code-js-keyword-operator
               ^
               source-code-js-whitespace
                ^
                source-code-js-literal-string-literal-string-begin
                 ^^^^^^^^^^^^^
                 source-code-js-literal-string-literal-string-unquoted
                              ^
                              source-code-js-literal-string-literal-string-end
                               ^
                               source-code-js-punctuation

// Print the message.
^^^^^^^^^^^^^^^^^^^^^
source-code-js-comment-line
console.log(`The message is: ${message}`);
^^^^^^^
source-code-js-identifier
       ^
       source-code-js-punctuation
        ^^^
        source-code-js-identifier
           ^
           source-code-js-punctuation
            ^
            source-code-js-literal-string-literal-string-begin
             ^^^^^^^^^^^^^^^^
             source-code-js-literal-string-literal-string-unquoted
                             ^^^^^^^^^^
                             source-code-js-literal-string-literal-string-unquoted-literal-string-interpolation
                                       ^
                                       source-code-js-literal-string-literal-string-end
                                        ^
                                        source-code-js-punctuation
                                         ^
                                         source-code-js-punctuation
